### ISSUE TITLE
Translation System On Crowdin

### ISSUE THREAD

**Issue by Minionguyjpro** [windows-server-2003/ThirdTube#101]
Put this project on Crowdin so users can translate the app and you have multiple languages!

**Comment by CSab6482** [comment]
Initially I thought you meant programming languages, but it appears you mean spoken languages. Therefore, would this ticket be a more generalized version of issue #99?

**Comment by Minionguyjpro** [comment]
> Initially I thought you meant programming languages, but it appears you mean spoken languages. Therefore, would this ticket be a more generalized version of issue #99?

You're right. I saw that issue and got the idea to just put it on Crowdin, let users translate their own language, and commit it for a new release.
